api/envoy/extensions/filters/udp/udp_proxy/v3/udp_proxy.proto

```diff
@@ -88,4 +89,8 @@ message UdpProxyConfig {
   // The default if not specified is false, that means each data chunk is forwarded
   // to upstream host selected on first chunk receival for that "session" (identified by source IP/port and local IP/port).
   bool use_per_packet_load_balancing = 7;
+
+  // Configuration for :ref:`Dynamic Metadata <config_access_log_format_dynamic_metadata>`
+  // emitted by the udp_proxy.
```

Comment:
The actual metadata needs to be documented. Please document each thing and then examples of the substitution string required to extract it so that users understand what is available. Thank you!

/wait

Reply:
> The actual metadata needs to be documented. Please document each thing and then examples of the substitution string required to extract it so that users understand what is available. Thank you!
> 
> /wait

OK, I have updated usage and added access log format example, please review. Thanks!